crates/biome_js_analyze/src/lint/nursery/no_static_element_interactions.rs

```diff
@@ -0,0 +1,353 @@
+    /// <span role="link" onClick={()=>{})}></span>
+    /// <a href="http://example.com" onClick={()=>{})}></a>
+    /// ```
+    ///
+    pub NoStaticElementInteractions {
+        version: "next",
+        name: "noStaticElementInteractions",
+        language: "js",
+        sources: &[RuleSource::EslintJsxA11y("no-static-element-interactions")],
+        recommended: false,
+    }
+}
+
+// These are interactions defined by eslint-plugin-jsx-a11y.
+// ref: https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/974275353598e9407c76bd4a50c331a953755cee/src/rules/no-static-element-interactions.js#L33-L37
+// ref: https://github.com/jsx-eslint/jsx-ast-utils/blob/main/src/eventHandlers.js
+lazy_static::lazy_static! {
+    static ref INTERACTIVE_HANDLER_LIST: HashMap<&'static str, Vec<&'static str>> = {
+        let mut m = HashMap::new();
+        m.insert("clipboard", vec!["onCopy", "onCut", "onPaste"]);
+        m.insert("composition", vec!["onCompositionEnd", "onCompositionStart", "onCompositionUpdate"]);
+        m.insert("keyboard", vec!["onKeyDown", "onKeyPress", "onKeyUp"]);
+        m.insert("focus", vec!["onFocus", "onBlur"]);
+        m.insert("form", vec!["onChange", "onInput", "onSubmit"]);
+        m.insert("mouse", vec![
+            "onClick", "onContextMenu", "onDblClick", "onDoubleClick", "onDrag", "onDragEnd",
+            "onDragEnter", "onDragExit", "onDragLeave", "onDragOver", "onDragStart", "onDrop",
+            "onMouseDown", "onMouseEnter", "onMouseLeave", "onMouseMove", "onMouseOut",
+            "onMouseOver", "onMouseUp"
+        ]);
+        m.insert("selection", vec!["onSelect"]);
+        m.insert("touch", vec!["onTouchCancel", "onTouchEnd", "onTouchMove", "onTouchStart"]);
+        m.insert("ui", vec!["onScroll"]);
+        m.insert("wheel", vec!["onWheel"]);
+        m.insert("media", vec![
+            "onAbort", "onCanPlay", "onCanPlayThrough", "onDurationChange", "onEmptied",
+            "onEncrypted", "onEnded", "onError", "onLoadedData", "onLoadedMetadata", "onLoadStart",
+            "onPause", "onPlay", "onPlaying", "onProgress", "onRateChange", "onSeeked", "onSeeking",
+            "onStalled", "onSuspend", "onTimeUpdate", "onVolumeChange", "onWaiting"
+        ]);
+        m.insert("image", vec!["onLoad", "onError"]);
+        m.insert("animation", vec!["onAnimationStart", "onAnimationEnd", "onAnimationIteration"]);
+        m.insert("transition", vec!["onTransitionEnd"]);
+        m
+    };
+}
+
+impl Rule for NoStaticElementInteractions {
+    type Query = Aria<AnyJsxElement>;
+    type State = ();
+    type Signals = Option<Self::State>;
+    type Options = ();
+
+    fn run(ctx: &RuleContext<Self>) -> Self::Signals {
+        let node = ctx.query();
+        let aria_roles = ctx.aria_roles();
+        let element_name = node.name().ok()?.as_jsx_name()?.value_token().ok()?;
+        let attributes = extract_attrs(&node.attributes());
+        let elm = element_name.text_trimmed();
```

Comment:
chore; need to change where this is used.
```suggestion
        let elment_name = element_name.text_trimmed();
```

Reply:
@unvalley 
fixed in this commit 82626b0ad1f2df1082031e8a9b4d141fa15654a9